source/kubernetes/cheatsheet.html.md

```diff
@@ -1,180 +1,115 @@
 ---
-title: EKS/Kubernetes GOV.UK cheatsheet
+title: Kubernetes/EKS cheatsheet
 layout: multipage_layout
 weight: 5
 ---
 
-# EKS/Kubernetes GOV.UK cheatsheet
+# Kubernetes/EKS cheatsheet for GOV.UK developers
 
-This is a quick summary of the most useful commands and guidance for interacting with GOV.UK's EKS platform.
+This page lists some example commands for interacting with the GOV.UK
+Kubernetes clusters.
```

Comment:
Nit: consistency with line width - some lines are wrapped and some aren't

Reply:
Fixed :)